fast-socket-installer/fast-socket-installer.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: nccl-fastsocket-installer
+  name: nccl-fastsocket-installer2
```

Comment:
Can we use a better name for this? Like 'nccl-fastsocket-manual-installer' or something.

Reply:
Yes we can